### ISSUE TITLE
Error at startup TimeManager 1.10.0 / mc 1.16

### ISSUE THREAD

**Issue by soydisa** [ArVdC/TimeManager#38]
![image](https://github.com/user-attachments/assets/bd81b1ab-9660-483c-abe7-4aac495698ca)

Hi, i get an error while loading the plugin. The server uses Java 16 and Minecraft Paper 1.16.5

**Comment by ArVdC** [comment]
I think I found how to fix it. Please try this version : [https://github.com/ArVdC/TimeManager/releases/tag/v1.10.1-b](https://github.com/ArVdC/TimeManager/releases/tag/v1.10.1-b)

**Comment by ArVdC** [comment]
Did you try ?

**Comment by soydisa** [comment]
Yes, Sorry. The plugin don't load, exactly like the 1.10.0.

**Comment by ArVdC** [comment]
My bad, I forgot to change a line of code. I've actualized the file. Please try again

**Comment by soydisa** [comment]
It still not work
![Screenshot_2024-12-13-14-02-57-126_com opera gx-edit](https://github.com/user-attachments/assets/ce87ed57-8284-499a-add6-c3b7b30bf81a)

**Comment by soydisa** [comment]
You fixed that?

**Comment by ArVdC** [comment]
Yes, at least I think so : [TimeManager v1.10.1](https://github.com/ArVdC/TimeManager/releases/tag/v1.10.1)
